test/duct/logger/timbre_test.clj

```diff
@@ -78,7 +79,30 @@
       (logger/log logger :report ::testing)
       (is (re-matches
            #"(?x)
-             \d\d-\d\d-\d\d\ \d\d:\d\d:\d\d\ [^\s]+\ 
-             REPORT\ \[duct\.logger\.timbre-test:78\]\ -\ 
+             \d\d-\d\d-\d\d\ \d\d:\d\d:\d\d\ [^\s]+\
+             REPORT\ \[duct\.logger\.timbre-test:78\]\ -\
              :duct\.logger\.timbre-test/testing\n"
            (slurp tempfile))))))
+
+(deftest restore-root-timbre-config-test
+  (testing "manages taoensso.timbre/*config* binding"
+    (let [init-binding tao/*config*
+          config {::logger/timbre {:level :info, :appenders {}}}]
+
+      (assert (not= init-binding config)
```

Comment:
Ok it's not intended as a test assertion like `is` though, but as an indication to the reader that the value we inherit from timbre is definitely different to our chosen config value.

For example `timbre/*config*` could in theory change its defaults to be the same as `config`, the assertion proves the environment is correct.

Regardless it's no biggie, probably not worth the line of code.  So happy to remove it.